modules/ipi-install-network-requirements.adoc

```diff
@@ -161,4 +161,4 @@ You can reconfigure the control plane nodes to act as NTP servers on disconnecte
 [id='network-requirements-out-of-band_{context}']
 == Port access for the out-of-band management IP address
 
-The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the provisioner during installation, the out-of-band management IP address must be granted access to port `80` on the bootstrap host and port `6180` on the {product-title} control plane hosts.
+The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the provisioner during installation, the out-of-band management IP address must be granted access to port `80` on the bootstrap host and port `6180` on the {product-title} control plane hosts. TLS port `6183` is required for virtual media installation, for example, via redfish.
```

Comment:
Init cap for Redfish?

Reply:
@adellape indeed, it IS capitalized. But I had to find that outside of docs.redhat.com (https://en.wikipedia.org/wiki/Redfish_(specification) ) because the only two instances I could find refer to the API which is all lower/sans mono. I will make it capitalized in this case, because it's a reference to the standards name.